### ISSUE TITLE
Kafka SASL listener listen only on localhost

### ISSUE THREAD

**Issue by omerlh** [testcontainers/testcontainers-node#582]
**Expected Behaviour**
Container running in the same network with Kafka can connect to Kafka using SASL listener.

**Actual Behaviour**
The SASL endpoint is only advertised using localhost port, not the Kafka container port:

**Testcontainer Logs**

```
testcontainers [DEBUG] [604daae4d06b] Executed command "kafka-configs --alter --bootstrap-server BROKER://604daae4d06b:9092 --entity-type brokers --entity-name 1 --add-config advertised.listeners=[PLAINTEXT://localhost:55831,SECURE://localhost:55830,BROKER://604daae4d06b:9092]"... +2s
  testcontainers [DEBUG] [604daae4d06b] Executing command "kafka-configs --alter --zookeeper localhost:2181 --entity-type users --entity-name admin --add-config SCRAM-SHA-512=[password=supersecure]"... +0ms
  testcontainers:exec [604daae4d06b] Warning: --zookeeper is deprecated and will be removed in a future version of Kafka. +951ms
  testcontainers:exec [604daae4d06b] Use --bootstrap-server instead to specify a broker to connect to. +0ms
```

You can see that 2 regular listeners are configured - PLAINTEXT for localhost, BROKER for container port, but only one for SECURE with localhost port.

**Steps to Reproduce**
Use any container and try to connect to Kafka from the container, not from the test code (like existing tests does) with SASL.

You'll notice this error:
```
 "message" : "[Producer clientId=producer-1] Connection to node 1 (localhost/127.0.0.1:55854) could not be established. Broker may not be available.",
```

**Environment Information**
- Operating System: MacOS
- Docker Version: Docker version 20.10.21, build baeda1f
- Node version: v18.12.0
- Testcontainers version: 9.6.0

**Comment by omerlh** [comment]
I tried to spent some time on fixing it but I keep hitting weird Kafka issues so I opened this issue to ask for some help :)

**Comment by cristianrgreco** [comment]
Hi @omerlh, thanks for raising the issue. I was able to reproduce it in test and get it fixed, do you want to have a look?

**Comment by omerlh** [comment]
This is not released as pre-lease, correct? Looking at the code I think it will work - it is reproducing the exact use case I had but I am not sure how I can test it

**Comment by cristianrgreco** [comment]
If you're able to test a beta build that'd be great, please give version 9.9.0-beta.0 a go 🙂

**Comment by omerlh** [comment]
Yes I can confirm it is working :) I am having SSL issues now (certificate is not for the right server name, this is easy to fix with the right certificates) - but it proves the connection worked! Thanks you!

**Comment by cristianrgreco** [comment]
Thank you for raising the issue and fast feedback 🙂 
I'll be merging the PR and creating a release, probably within next 30m, will update here when new version available.

**Comment by cristianrgreco** [comment]
Released in 9.9.0
